build.gradle

```
@@ -76,22 +73,46 @@ embulkPlugin {
     mainClass = "org.embulk.filter.expand_json.ExpandJsonFilterPlugin"
     category = "filter"
     type = "expand_json"
-    ignoreConflicts = [
-        // They conflict with embulk-core:0.10.6, but embulk-core will remove them from its dependencies.
-        // They are intentionally included in the plugin's dependencies to keep it working before and after the removal.
-        // Warning messages against them are marked to ignore explicitly here.
-        [ group: "com.fasterxml.jackson.core", module: "jackson-annotations" ],
-        [ group: "com.fasterxml.jackson.core", module: "jackson-core" ],
-        [ group: "com.fasterxml.jackson.core", module: "jackson-databind" ],
-    ]
 }
 
 publishing {
     publications {
-        embulkPluginMaven(MavenPublication) {  // Publish it with "publishEmbulkPluginMavenPublicationToMavenRepository".
+        maven(MavenPublication) {
+            groupId = project.group
+            artifactId = project.name
+
             from components.java  // Must be "components.java". The dependency modification works only for it.
+            // javadocJar and sourcesJar are added by java.withJavadocJar() and java.withSourcesJar() above.
+            // See: https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html
+
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                packaging "jar"
+
+                name = project.name
+                description = project.description
+                url = "https://github.com/civitaspo/embulk-filter-expand_json"
+
+                licenses {
+                    license {
+                        // http://central.sonatype.org/pages/requirements.html#license-information
+                        name = "MIT License"
+                        url = "https://www.opensource.org/licenses/mit-license.php"
+                    }
+                }
+
+                developers {
+                    // TODO: Set <developers> if released for Maven Central.
+                }
+
+                scm {
+                    connection = "scm:git:git://github.com/civitaspo/embulk-filter-expand_json.git"
+                    developerConnection = "scm:git:git@github.com:civitaspo/embulk-filter-expand_json.git"
+                    url = "https://github.com/civitaspo/embulk-filter-expand_json"
+                }
+            }
         }
     }
+
     repositories {
         // TODO: Configure some public Maven repo(s).
```

If you're willing to release its JARs to Maven Central, this part would be like below :

```suggestion
        maven {  // publishMavenPublicationToMavenCentralRepository
            name = "mavenCentral"
            if (project.version.endsWith("-SNAPSHOT")) {
                url "https://oss.sonatype.org/content/repositories/snapshots"
            } else {
                url "https://oss.sonatype.org/service/local/staging/deploy/maven2"
            }

            credentials {
                username = project.hasProperty("ossrhUsername") ? ossrhUsername : ""
                password = project.hasProperty("ossrhPassword") ? ossrhPassword : ""
            }
        }
```